docs/_build/validmind.html

```diff
@@ -102,17 +102,17 @@ <h3>Submodules</h3>
     <main class="pdoc">
```

Comment:
anything in 'docs/_build' is automatically generated and commited via github actions so you don't need to manually update :)

Reply:
removed